spec/models/signature_spec.rb

```diff
@@ -18,6 +18,7 @@
 #
 
 require 'rails_helper'
+require 'webmock/rspec'
```

Comment:
Could probably put this into spec_helper or rails_helper or a support file so the webmock rspec dsl is available to all our specs.